src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -1061,7 +1061,11 @@ private void updateNameReference(
               Node assign = IR.assign(getProp, nameRef.getFirstChild().detachFromParent());
               assign.setJSDocInfo(info);
               Node expr = IR.exprResult(assign).useSourceInfoIfMissingFromForTree(nameRef);
-              parent.replaceWith(expr);
```

Comment:
This line was definitely wrong - it's assuming a var statement only has one child. However, the suggested fix isn't correct either. Here's the really difficult test case that needs handled:

```js
(() => {
  var x = 1, y = x + 1, z = y + 1;
  module.exports.y = y;
})();
```

It should become:

```js
goog.provide('module$test');
/** @const */ var module$test = {};
(() => {
  var x = 1;
  module$test.y = x + 1;
  var z = module$test.y + 1;
})();
```

In the case where the var statement has more than one declaration and `nameRef` isn't the last child, we need to split the var statement apart into two distinct statements and inserted the renamed expression inbetween them.